benchmark/single-source/MapReduce.swift

```diff
@@ -133,7 +133,7 @@ public func run_MapReduceShortString(_ N: Int) {
 
 @inline(never)
 public func run_MapReduceClass(_ N: Int) {
-  let numbers = (0..<1000).map { NSDecimalNumber(value: $0) }
+  let numbers = (0..<1000).map { NSDecimalNumber(integerLiteral: $0) }
```

Comment:
Fixed in https://github.com/apple/swift/pull/7810/commits/20801a0586b7939eaa24345656975b1ee98414de.